src/locale/it.js

```diff
@@ -90,6 +90,8 @@ export default moment.defineLocale('it', {
         hh: '%d ore',
         d: 'un giorno',
         dd: '%d giorni',
+        w: 'una settimana',
+        ww: '%d settimane',
```

Comment:
Hi @afilini or @atorin can I ask you guys to have a look at two words in Italian in this PR to moment.js library and provide feedback. (I found you at Developers Italia repo, sorry for bothering you but I thought to give it a go)

Reply:
+ @zenlor maybe you can verify?

Reply:
LGTM

Reply:
Yes it's fine, nothing to complain about!